label_studio/projects/models.py

```diff
@@ -750,6 +766,21 @@ def get_all_storage_objects(self, type_='import'):
         self._storage_objects = storage_objects
         return storage_objects
 
+    def update_tasks_counters(self, queryset):
+        objs = []
```

Comment:
need to rewrite it using tasks.update(total_annotations=F('xxx') - F('yyy')), the current implementation is very slow

Reply:
Fixed